### PR TITLE
Ruby 2.3.0 Update

### PR DESCRIPTION
**Pull request by KierranM**
Updating the ruby packages to install Ruby 2.3.0 which is the current stable version

### CODE REVIEW

File: manual/ruby/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -23,18 +23,24 @@
...
+  $url64 = 'http://dl.bintray.com/oneclick/rubyinstaller/rubyinstaller-2.3.0-x64.exe?direct'
+  $checksum64 = 'd9e18b23821e998cb9bd230495092ddd'
 
   $rubyPath = join-path $binRoot $('ruby' + "$rubyFolder")
   $silentArgs = "/verysilent /dir=`"$rubyPath`" /tasks=`"assocfiles,modpath`""
 
   # Install-ChocolateyPackage "$packageId" 'exe' "$silentArgs" "$url" -checksum $checksum
 
-  Install-ChocolateyPackage "$packageId" 'exe' "$silentArgs" "$url" "$url64" -checksum $checksum -checksumType 'sha1' -checksum64 $checksum64 -checksumType64 'sha1'
+  Install-ChocolateyPackage "$packageId" 'exe' "$silentArgs" "$url" "$url64" -checksum $checksum -checksumType 'md5' -checksum64 $checksum64 -checksumType64 'md5'
```
Comment (ferventcoder) [inline manual/ruby/tools/chocolateyInstall.ps1]: What happened to the sha1 values?

Comment (KierranM) [inline manual/ruby/tools/chocolateyInstall.ps1]: Grabbed the latest downloads from [here](http://dl.bintray.com/oneclick/rubyinstaller/), they had the md5 with them. I can switch it back to SHA1 if you wish.

Comment (ferventcoder) [inline manual/ruby/tools/chocolateyInstall.ps1]: Sha1 or Sha256 both work.

**Comment by KierranM** [comment]
@ferventcoder I have switched the packages to using SHA256 hashes

**Comment by ferventcoder** [comment]
Pushed and awaiting moderation.

**Comment by ferventcoder** [comment]
Thanks for doing that!

**Pull request merged**